lib/ansible/modules/source_control/gitlab_project_variable.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: gitlab_project_variable
+short_description: Creates/updates/deletes Gitlab Projects Variables
```

Comment:
I think the official spelling is GitLab with a capital L.
```suggestion
short_description: Creates/updates/deletes GitLab Projects Variables
```

Reply:
There are more occurences of Gitlab without a captial L.